pkg/envoy/server.go

```diff
@@ -664,22 +716,27 @@ func getCiliumTLSContext(tls *policy.TLSContext) *cilium.TLSContext {
 	}
 }
 
-func GetEnvoyHTTPRules(certManager policy.CertificateManager, l7Rules *api.L7Rules, ns string) *cilium.HttpNetworkPolicyRules {
+func GetEnvoyHTTPRules(certManager policy.CertificateManager, l7Rules *api.L7Rules, ns string) (*cilium.HttpNetworkPolicyRules, bool) {
 	if len(l7Rules.HTTP) > 0 { // Just cautious. This should never be false.
+		canShortCircuit := true
```

Comment:
please add a comment describing what does this mean

Reply:
Will do, how about something like this:
```
// Assume none of the rules have side-effects so that rule evaluation can
// be stopped as soon as the first allowing rule is found. 'canShortCircuit'
// is set to 'false' if any rules with side effects are encountered, causing
// all the applicable rules to be evaluated instead.
```

Reply:
Added.